### ISSUE TITLE
Add some documentation to README

### ISSUE THREAD

**Issue by quentin-st** [MrMitch/WPF-Loaders#1]
It would actually be awesome to have some documentation about **how to implement** those shiny animations inside an existing project.

We would also love to know how to **customize** those (using the `IsIndeterminate`, `Thickness` and `ColorX` properties)!

That would be great!
![Right?](https://media1.giphy.com/media/2AilMg2L8rTAA/giphy.gif)

**Comment by MrMitch** [comment]
Yes yes yes, all those things are planned and they will be addressed as soon as I can find some time. 

For the time being, the [MainWindow.xaml](https://github.com/MrMitch/WPF-Loaders/blob/master/Loaders/MainWindow.xaml) file shows several different ways the controls can be customized.

As far as using them in an existing project, there are two options I  can think of, off the top of my head : 
    1. you can copy/paste the files from this repository into an existing project :cry: 
    2. I can look into packaging them in a NuGet package :+1: 

**SO MANY THINGS TO DO**

![THINGS](https://tinyobsessions.files.wordpress.com/2015/04/giphy-7.gif)

**Comment by quentin-st** [comment]
The NuGet package would be **great**.

![](http://fr.eonline.com/eol_images/Entire_Site/2014120/rs_245x169-140220093119-tumblr_m8f60225Kg1r76lino1_250.gif)

**Comment by MrMitch** [comment]
https://www.nuget.org/packages/MrMitch.Loaders/

``` ps1
PM> Install-Package MrMitch.Loaders
```

![EVERYBODY DANCE NOW](http://ak-hdl.buzzfed.com/static/enhanced/webdr03/2013/9/16/23/anigif_enhanced-buzz-6719-1379387872-0.gif)
